src/jdk.incubator.vector/share/classes/jdk/incubator/vector/Vector.java

```diff
@@ -3510,6 +3512,92 @@ public abstract void intoByteArray(byte[] a, int offset,
     public abstract void intoByteBuffer(ByteBuffer bb, int offset,
                                         ByteOrder bo, VectorMask<E> m);
 
+    /**
+     * Stores this vector into a {@linkplain MemorySegment memory segment}
+     * starting at an offset using explicit byte order.
+     * <p>
+     * Bytes are extracted from primitive lane elements according
+     * to the specified byte ordering.
+     * The lanes are stored according to their
+     * <a href="Vector.html#lane-order">memory ordering</a>.
+     * <p>
+     * This method behaves as if it calls
+     * {@link #intoMemorySegment(MemorySegment,long,ByteOrder,VectorMask)
+     * intoMemorySegment()} as follows:
+     * <pre>{@code
```

Comment:
you might want to revisit later to use code snippets

Reply:
Yes, that's a good point, we need to revisit all of the doc.